frontend/tsconfig.json

```diff
@@ -16,8 +16,9 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "strict": true,
+    "baseUrl": "./src",
     "paths": {
-      "~/*": ["./src/*"]
+      "~/*": ["./*"]
```

Comment:
was this change for a specific reason?, I didnt see it in the isue

Reply:
Yes, this is due to playwright having issues with the tsconfig.json, you can check the conversation here: https://redhat-internal.slack.com/archives/C05SMJ09DD2/p1695204667231359